src/UI/Art/SpriteArtist/MapSelectMenuArtist.java

```diff
@@ -174,23 +181,20 @@ private static void drawMapSelectMenu(Graphics g, MapSelectController gameSetup)
     SpriteUIUtils.drawImageCenteredOnPoint(g, miniMap, drawScale*miniMapCenterX, drawScale*miniMapCenterY, mmScale);
   }
 
-  /**
-   * Count and return the number of tiles of the provided type in the given MapInfo.
-   */
-  private static int countTiles(MapInfo mapInfo, TerrainType tileType)
+  private static void countCapturables(MapInfo mapInfo, Map<TerrainType, Integer> propCounts)
   {
-    int count = 0;
     for(int y = 0; y < mapInfo.getHeight(); ++y)
     {
       for(int x = 0; x < mapInfo.getWidth(); ++x)
       {
-        if(mapInfo.terrain[x][y] == tileType)
+        TerrainType tt = mapInfo.terrain[x][y];
+        if( tt.isCapturable() )
         {
-          ++count;
+          int oldCount = propCounts.getOrDefault(tt, 0);
+          propCounts.put(tt, ++oldCount);
```

Comment:
We didn't formerly show HQ/Lab counts, since that's determined by player count. I guess it isn't a _problem_ to show them, but it seems redundant.

Reply:
Lab counts != player count.
Also, you can technically just slap down neutral HQs or extra HQs...

Reply:
I mean. I guess. Neutral HQs are just cities though. The "objective" structures are also just much less indicative of what kind of map it is than the other things. Not gonna get too worked up about it either way though.

Reply:
Neutral HQs stay HQs when captured, which presents a liability paired with an extra terrain star.
Is that... good game design? Probably not. But it's not exactly equivalent to a city.

You're right that it's less likely to give a strong signal re: how the map plays.
I guess I could filter them.